### DISCUSSION TITLE
Setting up ao3downloader?

### DISCUSSION THREAD

**Discussion by strangedog1277** [nianeyna/ao3downloader#138] [General]
Hi, I've looked through the troubleshooting and I don't see the answer. I've already deleted and redownloaded ao3 downloader a few times. I had a later version of python but I've uninstalled it, so now I only have 3.9.2

I have: python 3.9.2
Windows 10

Have downloaded ao3downloader, unzipped, and double clicked ao3downloader.cmd. First I get Microsoft Defender telling me it's not letting it run, but I tell it to let it run. Then I get this:

Error: [WinError 5] Access is denied: 'C:\\WINDOWS\\system32\\venv'
The system cannot find the path specified.
Requirement already satisfied: pip in c:\users\hazel\appdata\local\programs\python\python39\lib\site-packages (24.0)
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
python: can't open file 'C:\WINDOWS\system32\ao3downloader.py': [Errno 2] No such file or directory
Press any key to continue . . .

What am I doing wrong? I'm sorry I don't know what I'm doing. Is it because it's looking for it under system32 when it's in users/downloads?

**Comment by nianeyna** [comment]
You're right, it does seem like the initialization script is for some reason trying to run itself inside system32 rather than the folder it's actually in, which of course won't work. It's not immediately obvious to me *why* it's doing that, though.

Some googling suggests it may be trying to run with elevated permissions which is automatically changing the working directory to system32. I'm kind of stabbing in the dark as to how to fix that, but if you could do me a favor - please try downloading the code from here:

https://github.com/nianeyna/ao3downloader/tree/windows-administrator-fix

instead of from the main page, and try running ao3downloader.cmd using that code. If it works, please let me know, if it doesn't work, please also let me know!

**Reply by strangedog1277** [reply]
Thanks for your reply! Ok I've downloaded the new code from the administrator link. Here's the result:

Collecting pip
  Using cached pip-24.0-py3-none-any.whl (2.1 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.2.3
    Uninstalling pip-20.2.3:
      Successfully uninstalled pip-20.2.3
Successfully installed pip-24.0
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
C:\Users\Hazel\AppData\Local\Programs\Python\Python39\python.exe: can't open file 'C:\Users\Hazel\AppData\Local\Temp\e76d5491-3821-4c2e-af1d-939259773647_ao3downloader-windows-administrator-fix.zip.647\ao3downloader-windows-administrator-fix\ao3downloader.py': [Errno 2] No such file or directory
Press any key to continue . . .

Could it be an issue of not being on the administrator account for the computer? I wonder if I can shut off windows defender, maybe that's the culprit. It brings up the defender box and I have to click run anyway each time.

**Comment by nianeyna** [comment]
Hm. Can you explain exactly what steps you took to unzip the zip file please?

**Reply by strangedog1277** [reply]
OH.
::facepalm::
Yeah ok it works now I unzipped the file.
SORRY!!! So sorry for wasting your time omg.

**Reply by nianeyna** [reply]
LOL it's fine. I will add a note to the readme about unzipping so that future users can benefit from your experience. 😂  Can I ask though, when it worked did you use the normal version or the alternative link I gave you? I'm wondering if I should merge the "fix" just in case, or if it's really quite useless haha

**Reply by strangedog1277** [reply]
I used the alternative version, but I can try the regular version, jussec lol

**Reply by strangedog1277** [reply]
Yeah the regular one works just as well. AHAHAH well I have idiot-proofed it for you. Um. 
Thanks for making this nice thing and sorry again for wasting your time!

**Reply by nianeyna** [reply]
Ha. no worries at all. I think I might merge the "fix" anyways since it doesn't seem to have *hurt* anything and it did actually de-obscure the problem you were having at least!
